app/views/users/show.html.erb

```diff
@@ -1,24 +1,136 @@
-<p class="fs-5 mt-4 mb-2">
-  <strong>Név:</strong>
-  <%= @user.name %>
-</p>
+<div class="row">
+  <div class="col-12">
+    <div class="card">
+      <div class="card-header text-center">
+        <span class="text-muted">
+          <svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" fill="currentColor" class="bi bi-person-circle" viewBox="0 0 16 16">
+          <path d="M11 6a3 3 0 1 1-6 0 3 3 0 0 1 6 0z"/>
+          <path fill-rule="evenodd" d="M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8zm8-7a7 7 0 0 0-5.468 11.37C3.242 11.226 4.805 10 8 10s4.757 1.225 5.468 2.37A7 7 0 0 0 8 1z"/>
+        </svg>
+        </span>
+      </div>
+      <div class="card-body text-center row justify-content-lg-center justify-content-start">
+        <div class="col-auto text-start">
+          <p>
+            <strong>Név: </strong><%= @user.name %>
+          </p>
+          <p>
+            <strong>Email: </strong><%= @user.email %>
+          </p>
+          <% if admin_signed_in? %>
+            <p>
+              <strong>Provider: </strong><%= @user.provider %>
+            </p>
+            <p>
+              <strong>Admin: </strong><%= @user.admin %>
+            </p>
+          <% end %>
+          <% if admin_signed_in? %>
+            <%= link_to 'Módosítás', force_edit_user_path(@user), class: "btn btn-primary" %>
+          <% else %>
+            <%= link_to 'Módosítás', edit_user_path(@user), class: "btn btn-primary" %>
+          <% end %>
+        </div>
+      </div>
+    </div>
+  </div>
 
-<p class="fs-5 my-2">
-  <strong>E-mail:</strong>
-  <%= @user.email %>
-</p>
 
-<p class="fs-5 my-2">
-  <strong>Admin:</strong>
-  <%= @user.admin %>
-</p>
+  <div class="col-12 mt-5">
+    <div class="card">
+      <div class="card-header text-center">
+        <h5>Regisztrált események</h5>
+      </div>
+      <div class="card-body">
+        <table class="table">
```

Comment:
![oie_dchO11Dz2GLw](https://user-images.githubusercontent.com/60111772/130370393-2044790c-0a87-486b-ab8c-55866acfe39b.png)

There's a bit of overflow. Could be mended like this:

1. Create a css class in globals:
```css
.table-overflow-x-scroll {
  display: block;
  overflow-x: scroll;
  max-width: 100%;
}
```
2. Wrap the table around with this class like this:
```html
<div class="table-overflow-x-scroll">
  <table class="table">
```

Reply:
I see the problem, but i think a better approach would be to use the Bootstrap built in `table-responsive-<breakpoint>` feature